docs/release-notes/8.4.4-4.md

```diff
@@ -0,0 +1,79 @@
+
+* Dropping the primary key and adding a new `AUTO_INCREMENT` column as the primary key in descending order using the `INPLACE` algorithm resulted in failure. (Bug #36658450)
+
+* Queries using a descending primary key with the `index_merge` optimization sometimes returned incorrect results, including missing rows. (Bug #106207, Bug #33767814)
+
+* When large transactions were being received and applied, and a request to stop the replication channel was made using STOP REPLICA, MySQL failed to stop the channel properly. As a result, it stopped processing channel commands. Additionally, the server shutdown process did not complete gracefully, requiring either the MySQL process to be killed or the host system to be restarted. (Bug #115966, Bug #37008345)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.4.4 release notes].
+
+## Improvements
+
+* [PS-9148](https://perconadev.atlassian.net/browse/PS-9148): Extends the [Data masking](..//data-masking-overview.md) with new additions from MySQL 8.3.0 [Enterprise Data Masking and De-Identification Component Variables](https://dev.mysql.com/doc/refman/8.4/en/data-masking-component-variables.html).
+
+## Bug fixes
+
+* [PS-9391](https://perconadev.atlassian.net/browse/PS-9391): The replication broke with the error `HA_ERR_KEY_NOT_FOUND` when the `slave_rows_search_algorithms` were set to `INDEX_SCAN,HASH_SCAN`.
+
+* [PS-9416](https://perconadev.atlassian.net/browse/PS-9416): The error messages from the Key Management Interoperability Protocol (KMIP) component were not descriptive.
+
+* [9509](https://perconadev.atlassian.net/browse/PS-9509): Percona Server stoped tracking the `global_connection_memory` when using `thread_handling='pool-of-threads'`.
+
+* [PS-9537](https://perconadev.atlassian.net/browse/PS-9537): When building a new component that used mysql_command_xxx services (such as `mysql_command_factory`, `mysql_command_query`, etc.), it was impossible to reuse the same connection to run multiple queries. This issue was observed with `SELECT` queries, but it may also apply to `INSERT`, `UPDATE`, and `DELETE` operations.
+
+* [PS-9542](https://perconadev.atlassian.net/browse/PS-9542): Added Clang-19 to Azure pipelines, and fixed the clang-19 compilation issues.
+
+* [PS-9551](https://perconadev.atlassian.net/browse/PS-9551): When building a new component that used mysql_command_xxx services (such as `mysql_command_factory`, `mysql_command_query`, etc.), a server exit was encountered when setting the `MYSQL_COMMAND_LOCAL_THD_HANDLE` option.
+
+* [PS-9611](https://perconadev.atlassian.net/browse/PS-9611): An assertion failure occurred during server shutdown: `!is_set() || m_can_overwrite_status`.
+
+* [PS-9612](https://perconadev.atlassian.net/browse/PS-9612): Percona Server build failed if more than 128 threads were available. Percona merged the fix form [MariaDB](https://github.com/MariaDB/server/commit/fd9f1638ea7).
```

Comment:
Percona merged the fix form 
should be
Percona merged the fix from

Reply:
fixed